modin/experimental/engines/omnisci_on_ray/io.py

```diff
@@ -186,6 +186,9 @@ def read_csv(
                 null_values=None,
                 true_values=None,
                 false_values=None,
+                # timestamp fields should be handled as strigs if parse_dates
```

Comment:
Why so?

Reply:
This is default pandas behavior - if `parse_dates` doesn't explicitly set columns for parsing, data file fields will be handled as strings.